mypy/options.py

```diff
@@ -310,7 +310,7 @@ def new_semantic_analyzer(self) -> bool:
     def snapshot(self) -> object:
         """Produce a comparable snapshot of this Option"""
         # Under mypyc, we don't have a __dict__, so we need to do worse things.
-        d = dict(getattr(self, '__dict__', ()))
+        d: Dict[str, Any] = dict(getattr(self, '__dict__', ()))
```

Comment:
Mypy was not happy about this line, because it was `Union[Dict[str, Any], Dict[<unknown>, <unknown>]]`